lib/ts/ink_queue.cc

```diff
@@ -257,11 +260,8 @@ malloc_new(InkFreeList *f)
 {
   void *newp = nullptr;
 
-  if (f->alignment) {
-    newp = ats_memalign(f->alignment, f->type_size);
-    if (f->advice && (INK_ALIGN((uint64_t)newp, ats_pagesize()) == (uint64_t)newp)) {
-      ats_madvise((caddr_t)newp, INK_ALIGN(f->type_size, f->alignment), f->advice);
-    }
+  if (IS_POWER_OF_2(f->alignment)) {
```

Comment:
Why are you checking the alignment to be a power of 2?

Reply:
Isn't the alignment supposed to be power of 2?

Reply:
I'll remove this then.